docs/skopeo-copy.1.md

```diff
@@ -132,7 +132,7 @@ skopeo  copy --encryption-key jwe:./public.key --encrypt-layer 1 oci:local_nginx
 ```
 
 ## SEE ALSO
-skopeo(1), podman-login(1), docker-login(1)
+skopeo(1), skopeo-login(1), docker-login(1), containers-auth.json(5), containers-signature(5), containers-transports(5)
```

Comment:
I don’t think the users of `copy` need to read the internal format of the signature file.  A link to `containers-policy.json(5)` would be useful, OTOH.

Reply:
I was just grep-ing for signature, and found it here, so removed and replaced with containers-policy.json